libs/vscode-ui/feature-task-execution-form/src/lib/task-execution-form.component.ts

```diff
@@ -170,11 +171,26 @@ export class TaskExecutionFormComponent implements OnInit, AfterViewChecked {
   ) {}
 
   ngOnInit() {
-    this.architectSubject.next(this.initialSchema);
+    const optionFilter = (option: Option) =>
+      !(
+        option.type === OptionType.Array &&
+        option.items &&
+        (option.items as string[]).length === 0
+      );
+
```

Comment:
I would say this filtering more likely belongs in `normalizeSchema`/`schemaToOptions`

Reply:
I was thinking that too, but I moved it here because I feel like this is a UI issue because we don't support those advanced array types. 

I can move it 😄

Reply:
yeah.. I guess it seems like a schema responsibility to me, and we'll have to figure out how to get the Schema to support that, which might be even harder now that we don't have it typed upstream.